docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -0,0 +1,204 @@
+
+#### Projet Settings configuration
+
+After activating the Feature, ensure to configure the new **default revision** field from the [Project Settings](/console/project-configuration/project-settings.md) selecting one of the already existing branches (namely Revisions).
+
+:::info
+If no default revision is specified, the Console will use `main` as the default revision.
+This revision will be an empty configuration if no other `main` branch was previously created on the Project.
+:::
+
+If you want the Project to use a [**pull-based deployment strategy**](/development_suite/deploy/gitops-based/index.md#advantages-of-pull-based-deployment), its configuration must contain the `strategy` property set to `pull` (the value for push-based deployments is `push`). Based on the selected strategy, the `runnerTool` may take different values. 
+
+:::note
+For now, the deployment strategy can only be set from the backoffice. Please ask the support of your Mia-Platform referent to change it.
+:::
+
+### Further adjustments
+
+Here are some **points of attention** and possible modifications needed to make sure everything works properly on the Git repository:
+
+- All the Console configuration state files that were previously saved on Git can be safely removed, this comprehends the following list of files and directories:
+  - `api-console-config.json`
+  - `fastdata-config.json`
+  - `mia-craft-config.json`
+  - `rbacManagerConfig.json`
+  - `backofficeConfigurations.json`
+  - `config-maps`
+  - `config-extension`
+  - `variables`
+  - `overlays/:env/variables.env`
+- If you have a pipeline file (e.g. `gitlab-ci.yml`) it may contain deployment scripts that may work based  the repository directory structure based on the previous directory structure. For this reason custom pipelines have to be updated according to the new directory structure; if your Project is using a pipeline provided by Mia-Platform templates a new template can be used, ask your Mia-Platform referent to know which one.
+- If you are **deploying without Kustomize**, you may find, within the `configuration` directory, a set of directories, one for each environment of your Project. You need to create the `overlays` directory, where you have to move environment directories previously stored inside `configuration`. No actions are required for the custom files in the `configuration` directory.
+- If you are **using Kustomize**, the `configuration` directory contains a `kustomization.yaml` manifest and possibly your custom files, which are already in the right spot. You can delete every other file inside the directory.
+
+    Also, make sure that: 
+    - Inside each `overlays/<environment>` directory, the `kustomization.yaml` manifest imports the corresponding manifest from the `environments/<environment>` directory, if you don't have the `environments` directory yet don't worry, it'll be created at your first deployment!
```

Comment:
```suggestion
    - Inside each `overlays/<environment>` directory, the `kustomization.yaml` manifest imports the corresponding manifest from the `environments/<environment>` directory. If you cannot find the `environments` directory yet, don't worry: it will be created at your first deployment!
```